src/types/type-declarations.ts

```diff
@@ -75,14 +79,15 @@ export interface GTag {
   (fn: 'config', opt: 'reset'): void;
 }
 
-export interface GoogleAnalyticsApi {
```

Comment:
this is technically a breaking change for the types

it's probably safe as i can't imagine anyone besides Nuxt Scripts using it but please be mindful of these changes down the line

Reply:
correct, Nuxt Script is currently the only framework using the types from here